sinch/domains/voice/models/calls/responses.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+from dataclasses import dataclass
+from sinch.core.models.base_model import SinchBaseModel
+from sinch.domains.voice.models import Price, Destination
+
+
+@dataclass
+class GetVoiceCallResponse(SinchBaseModel):
+    from_: str
+    to: Destination
+    domain: str
+    call_id: str
+    duration: int
+    status: str
+    result: str
+    reason: str
+    timestamp: datetime
+    custom: dict
```

Comment:
Custom seems to be a string https://developers.sinch.com/docs/voice/api-reference/voice/tag/Calls/#tag/Calls/operation/Calling_GetCallResult!c=200&path=custom&t=response

Reply:
fixed